Tests/Functional/SeleniumSessionTest.php

```diff
@@ -6,11 +6,11 @@
 
 /**
  * @group selenium
- */
+ *
```

Comment:
Well, removing the end of the phpdoc to make it comment the whole class is just wrong. If you don't want to run this test at all, remove it entirely